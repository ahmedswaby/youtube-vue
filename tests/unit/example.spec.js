import { mount, shallowMount } from "@vue/test-utils";
import ChannelPage from "@/views/channel.vue";

describe("ChannelPage.vue", () => {
    let wrapper = null;

  beforeEach(async () => {
		wrapper = mount(ChannelPage, { propsData: {} });
	});
  it("should match the snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});

describe('like a snapshot', () => {
  it('should return an h1', () => {
    let shallowWrapper = shallowMount(ChannelPage);
    expect(shallowWrapper.contains('h1')).toBe(true);


	});
});
