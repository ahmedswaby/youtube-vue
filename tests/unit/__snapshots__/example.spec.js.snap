// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChannelPage.vue should match the snapshot 1`] = `
<div>
  <h1>
    better This is an Channel Page
  </h1>
</div>
`;
